StatBot/StatBot/Program.cs

```diff
@@ -52,6 +53,11 @@ private Program()
                 CaseSensitiveCommands = false,
             });
 
+            // Initialize config object and config service
+            string path = Path.Combine(Directory.GetCurrentDirectory(), @"config.json");
```

Comment:
nit: `@` is pointless here, there are no escaped characters in your string.

Reply:
Forgot to remove that from when it did have escape chars